.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js: stable
 sudo: false
+before_install: "curl -L https://unpkg.com/@pnpm/self-installer | node"
```

Comment:
What's the reason to use `pnpm`? I think, it makes things more complicated where unnecessary

Reply:
[pnpm](https://pnpm.js.org) is the third largest node package manager, which uses symlinks and isolated trees to&nbsp;prevent code from accessing packages which they haven’t declared in their dependencies.

**See also:**
- https://medium.com/pnpm/pnpms-strictness-helps-to-avoid-silly-bugs-9a15fb306308
- https://www.kochan.io/nodejs/why-should-we-use-pnpm.html

Reply:
Linking the related conversation in mdn/mdn here: https://github.com/mdn/mdn/pull/69